README.md

```diff
@@ -2,6 +2,8 @@
 
 Eraser helps Kubernetes admins remove a list of non-running images from all Kubernetes nodes in a cluster.
 
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/{owner}/{repo}/badge)](https://api.securityscorecards.dev/projects/github.com/{owner}/{repo})
```

Comment:
Not sure if this is just because of how it's generated, but when I check the preview it looks like this:
![scorecard](https://user-images.githubusercontent.com/36644727/231010279-5dd74c85-25b9-4311-aba2-ec069afefe85.png)

Reply:
i think it'll be visible when it's merged. example: https://github.com/ossf/scorecard/blob/1fb59608bd73a5bfc0c1821bb1b36c7a4afcad29/README.md?plain=1#L162